toolchains/atari/Dockerfile.m4

```diff
@@ -1,15 +1,21 @@
 m4_include(`paths.m4')m4_dnl
-m4_include(`packages.m4')m4_dnl
```

Comment:
Why is this being replaced with basically what is the content of `packages.m4` ? Couldn't just `local_sdk_package` be added?